src/player_activity.cpp

```diff
@@ -34,26 +34,27 @@ player_activity::player_activity( activity_type t, int turns, int Index, int pos
 
 const std::string &player_activity::get_stop_phrase() const
 {
-    static const std::string stop_phrase[NUM_ACTIVITIES] = {
-        _( " Stop?" ), _( " Stop reloading?" ),
-        _( " Stop reading?" ), _( " Stop playing?" ),
-        _( " Stop waiting?" ), _( " Stop crafting?" ),
-        _( " Stop crafting?" ), _( " Stop disassembly?" ),
-        _( " Stop butchering?" ), _( " Stop salvaging?" ), _( " Stop foraging?" ),
-        _( " Stop construction?" ), _( " Stop interacting with the vehicle?" ),
-        _( " Stop pumping gas?" ), _( " Stop training?" ),
-        _( " Stop waiting?" ), _( " Stop using first aid?" ),
-        _( " Stop fishing?" ), _( " Stop mining?" ), _( " Stop burrowing?" ),
-        _( " Stop smashing?" ), _( " Stop de-stressing?" ),
-        _( " Stop cutting tissues?" ), _( " Stop dropping?" ),
-        _( " Stop stashing?" ), _( " Stop picking up?" ),
-        _( " Stop moving items?" ), _( " Stop interacting with inventory?" ),
-        _( " Stop fiddling with your clothes?" ), _( " Stop lighting the fire?" ),
-        _( " Stop working the winch?" ), _( " Stop filling the container?" ),
-        _( " Stop hotwiring the vehicle?" ), _( " Stop aiming?" ),
-        _( " Stop using the ATM?" ), _( " Stop trying to start the vehicle?" ),
-        _( " Stop welding?" ), _( " Stop cracking?" ), _( " Stop repairing?" ),
-        _( " Stop modifying gun?" )
+    static const std::array<std::string, NUM_ACTIVITIES> stop_phrase = {{
+            _( " Stop?" ), _( " Stop reloading?" ),
```

Comment:
Btw. This looks like one of those whitespace in translation issues. The phrase from here is appended to another string to be displayed in the yes-no query. The code should insert the single space there and not require it to be part of the original phrase.